modules/bullet/bullet_physics_server.cpp

```diff
@@ -1519,7 +1523,7 @@ void BulletPhysicsServer::free(RID p_rid) {
 		space_owner.free(p_rid);
 		bulletdelete(space);
 	} else {
-		ERR_FAIL_MSG("Invalid ID.");
+		ERR_FAIL_MSG("Invalid RID.");
```

Comment:
This should be removed if we go with the silent fail like on VisualServer. There's another instance elsewhere.